src/Compute/Compute/Extension/AzureVMBackup/SetAzureVMBackupExtension.cs

```diff
@@ -61,6 +61,9 @@ public class SetAzureVMBackupExtension : VirtualMachineExtensionBaseCmdlet
             HelpMessage = "The tag for this backup.")]
         public string Tag { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Returns immediately with status of request")]
```

Comment:
Revert this one, since it calls the backup method above.

Reply:
Done